docs/index.md

```diff
@@ -64,10 +64,14 @@ enablePlugins(com.mpc.scalats.sbt.TypeScriptGeneratorPlugin)
 Currently *scala-ts* supports the following types of case class members:
 
 * `Int`, `Double`, `Boolean`, `String`, `Long`
-* `List`, `Seq`, `Set`
-* `Option`
-* `LocalDate`, `Instant`, `Timestamp`
+* `List`, `Seq`, `Set`, `Map`
+* `Option`, `Either`
+* `LocalDate`, `LocalDateTime`, `Instant`, `Timestamp`, `ZonedDateTime`
+* `BigDecimal`
```

Comment:
Revert

Reply:
Let's resolve the above discussions related to `BigDecimal`/`UUID` first